app/src/main/java/one/mixin/android/ui/common/messenger/Messenger.kt

```diff
@@ -208,8 +208,8 @@ class Messenger @Inject internal constructor(private val jobManager: MixinJobMan
         val transferContactData = ContactMessagePayload(shareUserId)
         val encoded = GsonHelper.customGson.toJson(transferContactData).base64Encode()
         val message = createContactMessage(
-            UUID.randomUUID().toString(), conversationId, sender.userId,
-            category, encoded, shareUserId, MessageStatus.SENDING.name, nowInUtc(), replyMessage?.messageId, replyMessage?.toQuoteMessageItem()
+            UUID.randomUUID().toString(), conversationId, sender.userId, category, encoded, shareUserId,
+                MessageStatus.SENDING.name, nowInUtc(), sender.fullName, replyMessage?.messageId, replyMessage?.toQuoteMessageItem()
```

Comment:
Here you should insert contact data,  not sender.

Reply:
fixed.